Chapters/04-implicitni-funkce.tex

```diff
@@ -50,6 +50,83 @@ \subsection{Věty o implicitní funkci}
 	$f: (x_0 - \delta , x_0 + \delta ) \to \mathbb{R}$ má spojité derivace do řádu $k$.
 \end{theorem}
 
+\begin{proof}
+	Buď třeba
+	$$
+	\frac{\partial F(x_0, y_0)}{\partial y} > 0
+	$$
+	Vzhledem ke spojitosti existují $\delta_1, \Delta > 0$ taková, že v
+	obdélníku $\langle x_0 - \delta_1, x_0 + \delta_1 \rangle \times
+	\langle y_0 - \Delta, y_0 + \Delta \rangle$ je $\frac{\partial
+	F}{\partial y} (x, y)$ stále kladná (víme, že $\exists \epsilon:
+	\frac{\partial F(x_0, y_0)}{\partial y} > \epsilon > 0$ a pro toto
+	$\epsilon$ mi spojitost PD dá $\delta_1, \Delta$). Tento obdélník je
+	kompaktní, tedy na něm spojité funkce nabývají extrémů a tedy $\exists
+	a, A > 0:$
+	\begin{equation} \label{eq:1}
+		\frac{\partial F}{\partial y}(x, y) > a,~ \left |
+		\frac{\partial F}{\partial x}(x, y) \right | < A
+	\end{equation}
+	Vezměme funkci $\varphi(y) = F(x_0, y)$. Ta má kladnou derivaci a tedy
+	roste na intervalu $\langle y_0 - \Delta, y_0 + \Delta \rangle$.
+	Jelikož $\varphi(y_0) = 0$, je $F(x_0, y_0 - \Delta) = \varphi(y_0 -
+	\Delta) < 0$ a $F(x_0, y_0 + \Delta) > 0$. Jelikož funkce $F$ je
+	spojitá (má spojité parciální derivace a tedy i totální diferenciál),
+	existuje $\delta > 0$ takové, že $F(x, y_0 - \Delta) < 0$ a $F(x, y_0 +
+	\Delta) > 0$ pro všechna $x \in (x_0 - \delta, x_0 + \delta)$. Pro
+	taková $x$ položme $\varphi_x(y) = F(x, y)$. Máme vždy $\varphi_x'(y) >
+	0$ na celém $\langle y_0 - \Delta, y_0 + \Delta \rangle$, takže je tam
+	každá z těchto funkcí rostoucí a ovšem spojitá. Jelikož $\varphi_x(y_0
+	- \Delta) < 0 < \varphi(y_0 + \Delta)$, nabývá $F(x, y) =
+	\varphi_x(y)$, v intervalu $(y_0 - \Delta, y_0 + \Delta)$ nuly v právě
+	jednom bodě $y$. Označme toto $y$ jako $f(x)$.
+
+	Zatím však ani nevíme, zda je $f$ spojitá. Pojďme odvodit vlastnosti
+	$f$.
+	$$
```

Comment:
I'd have to look into this more deeply, but wouldn't it be better to use something like an `align*` block instead?